addons/website_sale/models/res_config_settings.py

```diff
@@ -92,6 +92,13 @@ class ResConfigSettings(models.TransientModel):
     enabled_extra_checkout_step = fields.Boolean(string="Extra Step During Checkout")
     enabled_buy_now_button = fields.Boolean(string="Buy Now")
 
+    selectable_pricelist_ids = fields.Many2many(
+        comodel_name='product.pricelist',
+        relation='selectable_pricelists',
```

Comment:
```suggestion
```

Is the relation necessary? it won't be stored on the wizard model anyway :thinking: 
Also, the comodel_name is automatically deducted from the `related` field.